spec.md

```diff
@@ -763,13 +756,87 @@ For success responses, the following fields are defined:
+deprovision request to the Service Broker to prevent an orphan being created on
+the Service Broker. However, while the Platform will attempt
 to send a deprovision request, Service Brokers MAY automatically delete
 any resources associated with the failed bind request on their own.
 
-### Polling Interval and Duration
+## Polling Last Operation for Service Bindings
+
+When a broker returns status code `202 Accepted` for [Binding](#binding) or
+[Unbinding](#unbinding), the Platform will begin polling the
+`/v2/service_instances/:instance_id/service_bindings/:binding_id/last_operation`
+endpoint to obtain the state of the last requested operation.
+
+Returning `"state": "succeeded"` or `"state": "failed"` will cause the Platform
+to cease polling and, in the case of a [Binding](#binding) request, information
+regarding the Service Binding can then immediately be fetched using the
+[Fetching a Service Binding](#fetching-a-service-binding) endpoint.
+
+### Request
+
+#### Route
+`GET /v2/service_instances/:instance_id/service_bindings/:binding_id/last_operation`
+
+`:instance_id` MUST be a globally unique non-empty string.
+`:binding_id` MUST be a globally unique non-empty string.
+
+#### Parameters
+
+The request provides these query string parameters as useful hints for brokers.
+
+| Query-String Field | Type | Description |
+| --- | --- | --- |
+| service_id | string | ID of the service from the catalog. If present, MUST be a non-empty string. |
+| plan_id | string | ID of the plan from the catalog. If present, MUST be a non-empty string. |
+| operation | string | A broker-provided identifier for the operation. When a value for `operation` is included with asynchronous responses for [Binding](#binding) and [Unbinding](#unbinding) requests, the Platform MUST provide the same value using this query parameter as a URL-encoded string. If brokers do not return this `operation` field, only one asynchronous operation MAY be supported at a time. If present, MUST be a non-empty string. |
+
+Note: Although the request query parameters `service_id` and `plan_id` are not
+mandatory, the Platform SHOULD include them on all `last_operation` requests it
+makes to Service Brokers.
+
+#### cURL
+```
+$ curl http://username:password@broker-url/v2/service_instances/:instance_id/service_bindings/:binding_id/last_operation
+```
+
+### Response
+
+| Status Code | Description |
+| --- | --- |
+| 200 OK | MUST be returned upon successful processing of this request. The expected response body is below. |
+| 400 Bad Request | MUST be returned if the request is malformed or missing mandatory data. |
+| 410 Gone | Appropriate only for asynchronous delete operations. The Platform MUST consider this response a success and remove the resource from its database. The expected response body is `{}`. Returning this while the Platform is polling for [Binding](#binding) operations SHOULD be interpreted as an invalid response and the Platform SHOULD continue polling. |
```

Comment:
`Returning this while the Platform is polling for [Binding](#binding) operations SHOULD be interpreted as an invalid response and the Platform SHOULD continue polling.`

Should be 

`Returning this while the Platform is polling for create operations SHOULD be interpreted as an invalid response and the Platform SHOULD continue polling.`